src/routes/v1/group/list.js

```diff
@@ -3,25 +3,23 @@ import Group from '../../../models/group'
 import User from '../../../models/user'
 
 const router = express.Router()
-
 // List for groups that the user is a member of
 router.post('/', async (req, res) => {
   try {
     const groupList = await Group.listUserGroups(req.body.userId)
-
-    const fetchedGroup = groupList.map((group, i) => {
-      const ownerId = group.owner
-      const owner = User.getNameById(ownerId)
-      const ownerName = owner.profile.fullname
-
-      return {...group, owner: ownerName}
-    })
+    for (var index = 0; index < groupList.length; index++) {
+      var ownerId = groupList[index].owner
```

Comment:
Security rules for Generic Object Injection Sink `eslint(security/detect-object-injection)` failed